src/main/java/frc/robot/subsystems/elevator/ElevatorConstants.java

```diff
@@ -11,12 +11,19 @@ public class ElevatorConstants { // TODO: check real values
     public static final double GEAR_RATIO = 12.0;
     public static final double DRUM_RADIUS = 0.02; // [m]
 
+    // TODO: replace with actual values
     public static final MutableMeasure<Distance> STARTING_HEIGHT =
             Units.Meters.of(0).mutableCopy(); // [m]
     public static final MutableMeasure<Distance> MIN_HEIGHT =
             Units.Meters.of(0).mutableCopy(); // [m]
     public static final MutableMeasure<Distance> MAX_HEIGHT =
             Units.Meters.of(0).mutableCopy(); // [m]
+    public static final MutableMeasure<Distance> STARTING_CLIMB_HEIGHT =
+            Units.Meters.of(0).mutableCopy();
+    public static final MutableMeasure<Distance> ENDING_CLIMB_HEIGHT =
+            Units.Meters.of(0).mutableCopy();
+    public static final MutableMeasure<Distance> OUTTAKE_HEIGHT =
+            Units.Meters.of(0).mutableCopy(); // [m]
```

Comment:
```suggestion
            Units.Meters.of(0).mutableCopy();
```
Units in comments are unnecessary with `Measure` objects.